public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>My test page</title>
+</head>
+
+<body>
+  <div id='overview'></div>
+</body>
+<script type="module" src='../dist/overview.js'></script>
+<script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
```

Comment:
Is there an advantage to using a source script tag dependency vs importing/npm installing it?

Reply:
I believe it has to do with saving storage space? but this is a good question that I am not 100% sure what the answer is.